### PR TITLE
Monster.com scrapper added

### PR DESCRIPTION
**Pull request by zedunaid**
A scrapper that fetches jobs from monster.com is created.

### CODE REVIEW

**Comment by sak007** [comment]
Sync your branch with main. Failing checks will be fixed then.

File: requirements.txt [hunk 1]
```diff
@@ -57,5 +57,4 @@ virtualenv==20.0.17
 wadllib==1.3.3
 webdriver-manager==3.4.2
 zipp==1.0.0
-bs4
-schedule
```
Comment (sak007) [inline requirements.txt:61]: don't remove schedule from requirements

Comment (zedunaid) [reply requirements.txt:61]: Okay

**Review by sak007 — APPROVED** [review]

**Review by sak1997 — APPROVED** [review]
Looks good

**Pull request merged**